pkg/injector/envoy_config.go

```diff
@@ -130,6 +125,23 @@ func getProbeResources(config envoyBootstrapConfigMeta) ([]*xds_listener.Listene
 	return listeners, clusters, nil
 }
 
+func (wh *mutatingWebhook) createEnvoyBootstrapFromExisting(newName, oldName, namespace string, cert *certificate.Certificate) (*corev1.Secret, error) {
```

Comment:
As the function name doesn't indicate what "Existing" is, either rename it or add a function comment. Additionally, `s/newName/newBootstrapSecretName`, `s/oldName/oldBootstrapSecretName`.